src/interpreter/index.ts

```diff
@@ -184,6 +184,33 @@ export class Interpreter implements Expr.Visitor<BrsType>, Stmt.Visitor<BrsType>
         }
     }
 
+    callNamedFunction(functionName: string) {
```

Comment:
I'm still concerned / confused about how this will find the correct named function for more complex components but this looks sweet for single files.

Reply:
Great point!  It almost seems like the callable gets found immediately when `observeField` is called, and the list of observers winds up being a reference to each of those?